service-wfs-client/src/main/java/org/oskari/service/wfs/client/WFSFilter.java

```diff
@@ -0,0 +1,213 @@
+package org.oskari.service.wfs.client;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import fi.nls.oskari.service.ServiceRuntimeException;
+import org.geotools.factory.CommonFactoryFinder;
+import org.opengis.filter.Filter;
+import org.opengis.filter.FilterFactory;
+import org.opengis.filter.expression.Expression;
+
+public class WFSFilter {
```

Comment:
Rename this to OskariWFSFilter so it's clear that this is us and not for example GeoTools. Maybe throw in a comment that this is used for tranlating the Oskari specific JSON filter to a GeoTools filter.